src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationRepository.cs

```diff
@@ -199,12 +200,27 @@ public async Task<ICollection<Organization>> GetAddableToProviderByUserIdAsync(
 
     public async Task<ICollection<Organization>> GetManyByIdsAsync(IEnumerable<Guid> ids)
     {
-        await using var connection = new SqlConnection(ConnectionString);
+        using (var connection = new SqlConnection(ConnectionString))
+        {
+            var results = await connection.QueryAsync<Organization>(
+                "[dbo].[Organization_ReadManyByIds]",
+                new { OrganizationIds = ids.ToGuidIdArrayTVP() },
+                commandType: CommandType.StoredProcedure);
 
-        return (await connection.QueryAsync<Organization>(
-            $"[{Schema}].[{Table}_ReadManyByIds]",
-            new { OrganizationIds = ids.ToGuidIdArrayTVP() },
-            commandType: CommandType.StoredProcedure))
-            .ToList();
+            return results.ToList();
+        }
+    }
```

Comment:
Unrelated changes to this method - please revert.

Reply:
revert